### PR TITLE
Change in DB 

### PR DESCRIPTION
**Pull request by anhvu202**
Some attributes of "Orders" and "OrderDetails" in Database were changed.
The queries were updated.
Btw i missnamed the branch and have no idea how to change it :)

### CODE REVIEW

File: Application/app.py [hunk 1]
```diff
@@ -97,22 +97,204 @@ def delete(self):
...
+class OrderDetail(Resource):
+    def get(self):
+        orderDetails = getAllOrderDetails(engine)
+        return jsonify(orderDetails)
+
+    def post(self):
+        args = orderDetail_post_args.parse_args()
+        addOrderDetail = engine.execute("\
+            INSERT INTO order (orders_id, product_id, quantity, price)\
+                VALUES (%s, %s, %s, %s)", (args["orderId"], args["productId"],args["quantity"], args["price"]))
```
Comment (bila9630) [inline Application/app.py:293]: here also :)

File: Application/query.py [hunk 2]
```diff
@@ -9,6 +13,25 @@ def getAllSuppliers(engine):
...
+        result.append(newRow)
+
+    return result
+
+def getAllProducts(engine):
+    products = engine.execute("SELECT * FROM product")
+    result = []
+    for row in products:
+        newRow = {"id": row[0], "product_name": row[1], "quantity": row[2],
+                  "selling_price": row[3]}
```
Comment (bila9630) [inline Application/query.py:32]: I would format it, so there are two pairs on each row :)

File: Application/query.py [hunk 3]
```diff
@@ -20,3 +43,35 @@ def getAllIngredients(engine):
...
+        result.append(newRow)
+    
+    return result
+
+def getAllOrders(engine):
+    orders = engine.execute("SELECT * FROM order")
+    result = []
+
+    for row in orders:
+        newRow = {"id": row[0], "customerId": row[1], "customerName": row[2], "date": row[3], "shipTo": row[4], "invoiceAmount": row[5]}
```
Comment (bila9630) [inline Application/query.py:63]: here also

**Comment by bila9630** [comment]
Great work! 💯  Did you test those endpoints? If everything works, we can merge then :)

**Comment by anhvu202** [comment]
> Great work! 💯 Did you test those endpoints? If everything works, we can merge then :)

Everything works well... except for Order & OrderDetail. I'm not sure about it :)

**Pull request merged**